alf/batch_norm.py

```diff
@@ -317,9 +317,9 @@ def prepare_rnn_batch_norm(module: nn.Module) -> bool:
         if isinstance(m, (BatchNorm1d, BatchNorm2d)):
             bns.add(m)
         elif isinstance(m, (nn.BatchNorm1d, nn.BatchNorm2d, nn.BatchNorm3d)):
-            raise ValueError(
-                "RNN does not support torch.nn.BatchNorm layer "
-                "(at %s). Please use alf.layers.BatchNorm instead." % path)
+            warning_once(
+                "MCTSModel may not perform well with torch.nn.BatchNorm layer "
```

Comment:
I think it might be more general to mention RNN instead of `MCTSModel` when making this recommendation.

Reply:
agree.